gsgmail_process_email.py

```diff
@@ -1086,6 +1086,19 @@ def _set_sdi(self, input_dict):
 
         return phantom.APP_SUCCESS
 
+    def _get_fips_enabled(self):
+        try:
+            from phantom_common.install_info import is_fips_enabled
+        except ImportError:
+            return False
+
+        fips_enabled = is_fips_enabled()
+        if fips_enabled:
+            self.debug_print('fips is enabled')
+        else:
+            self.debug_print('fips is not enabled')
+        return fips_enabled
```

Comment:
Help is taken from https://github.com/splunk-soar-connectors/imap/pull/1/files#diff-f1d05048a6b2999ac6c73906bb26fbd987b94b3e8db082c1b10e29f25dba6e3aR218